### PR TITLE
feat: Create default main export for gamut-styles [WEB-895]

### PR DESCRIPTION
**Pull request by jakemhiller**
## Overview
Set main file for gamut-styles to a new root index.js file

It was previously set to the utils SCSS file, but was never used.

```js
import { colors } from '@codecademy/gamut-styles';
```

### PR Checklist

- [ ] Related to Abstract designs:
- [ ] Related to JIRA ticket: [WEB-895]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### Description

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.eed859.0<br/>@codecademy/gamut-styles@3.3.2-alpha.eed859.0<br/>@codecademy/gamut@8.2.2-alpha.eed859.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.c39ac2.0<br/>@codecademy/gamut-styles@3.3.2-alpha.c39ac2.0<br/>@codecademy/gamut@8.2.2-alpha.c39ac2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5eceeaec23f74fd9c7842acb--codecademy-styleguide.netlify.app>https://5eceeaec23f74fd9c7842acb--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5eceeaec23f74fd9c7842acb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.1d02c6.0<br/>@codecademy/gamut-styles@3.3.2-alpha.1d02c6.0<br/>@codecademy/gamut@8.2.2-alpha.1d02c6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ecfd06b560432f9b5d116a5--codecademy-styleguide.netlify.app>https://5ecfd06b560432f9b5d116a5--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ecfd06b560432f9b5d116a5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.b92ddf.0<br/>@codecademy/gamut-styles@3.3.2-alpha.b92ddf.0<br/>@codecademy/gamut@8.2.2-alpha.b92ddf.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ecfd819a8100c79034ae02d--codecademy-styleguide.netlify.app>https://5ecfd819a8100c79034ae02d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ecfd819a8100c79034ae02d>Deploy Logs</a></p>

File: packages/gamut-styles/utils/variables/index.js [hunk 1]
```diff
@@ -1,234 +1,7 @@
...
-  black: '#000000',
-  white: '#ffffff',
-  beige: '#efd9ca',
-  royalBlue: '#6400e4',
-};
+if (process.env.NODE_ENV === 'development') {
+  console.error(
+    'Importing from `gamut-styles/utils/variables` is now deprecated, please import directly from the gamut-styles package'
+  );
+}
```
Comment (jakemhiller) [inline packages/gamut-styles/utils/variables/index.js:5]: idk about this strategy, but it does prevent it from being a totally breaking change, happy to remove and make this breaking.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.cec4a5.0<br/>@codecademy/gamut-styles@3.3.2-alpha.cec4a5.0<br/>@codecademy/gamut@8.2.2-alpha.cec4a5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ecff78b6d4e3c0185975b77--codecademy-styleguide.netlify.app>https://5ecff78b6d4e3c0185975b77--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ecff78b6d4e3c0185975b77>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.97d1e1.0<br/>@codecademy/gamut-styles@3.3.2-alpha.97d1e1.0<br/>@codecademy/gamut@8.2.2-alpha.97d1e1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ecffa45f6b77c7e25003c7b--codecademy-styleguide.netlify.app>https://5ecffa45f6b77c7e25003c7b--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ecffa45f6b77c7e25003c7b>Deploy Logs</a></p>

File: packages/gamut-styles/utils/variables/index.js (hunk 1 above)
Comment (codecaaron) [reply packages/gamut-styles/utils/variables/index.js:5]: A convention I see for deprecations is generally `console.warn`, how does that feel for this?

I'm definitely cool with this strategy :)

Comment (jakemhiller) [reply packages/gamut-styles/utils/variables/index.js:5]: yeah, makes sense to me

File: packages/gamut-styles/utils/variables/index.js [hunk 2]
```diff
@@ -1,234 +1,7 @@
...
-    '800': '#323233',
-    '900': '#19191a',
-  },
-  black: '#000000',
-  white: '#ffffff',
-  beige: '#efd9ca',
-  royalBlue: '#6400e4',
-};
+if (process.env.NODE_ENV === 'development') {
+  console.error(
```
Comment (jakemhiller) [inline packages/gamut-styles/utils/variables/index.js:2]:
```suggestion
  console.warn(
```

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.13-alpha.1b0826.0<br/>@codecademy/gamut-styles@3.3.2-alpha.1b0826.0<br/>@codecademy/gamut@8.2.2-alpha.1b0826.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ee272af3f7b475f4489eeda--codecademy-styleguide.netlify.app>https://5ee272af3f7b475f4489eeda--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ee272af3f7b475f4489eeda>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.0-alpha.e1182f.0<br/>@codecademy/gamut-styles@4.2.0-alpha.e1182f.0<br/>@codecademy/gamut@12.1.0-alpha.e1182f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1f101d60d97f9b282d300c--gamut-preview.netlify.app>https://5f1f101d60d97f9b282d300c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f1f101d60d97f9b282d300c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.0-alpha.a6fc11.0<br/>@codecademy/gamut-styles@4.2.0-alpha.a6fc11.0<br/>@codecademy/gamut@12.1.0-alpha.a6fc11.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1f170c7ed8d6a432c86a0c--gamut-preview.netlify.app>https://5f1f170c7ed8d6a432c86a0c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f1f170c7ed8d6a432c86a0c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.0-alpha.c8f973.0<br/>@codecademy/gamut-styles@4.2.0-alpha.c8f973.0<br/>@codecademy/gamut@12.1.0-alpha.c8f973.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1f1d246f55790d614f98d3--gamut-preview.netlify.app>https://5f1f1d246f55790d614f98d3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f1f1d246f55790d614f98d3>Deploy Logs</a></p>

**Pull request merged**